README.md

```diff
@@ -2,15 +2,21 @@
 This repo aims to collect OSS made in Sudan or by Sudanese.
 Please feel free to submit a PR with your OSS (Package, App ...etc.), or OSS from someone you know.
 
-# Template:
+> For Contribute read [Contributing Guide](contributing.md) please
 
-- [Open Source Project Name](https://github.com/obiefy/api-response) - Small Description about the Open Source Project.
+## Plugins
 
-# Projects List:
 - [API Response](https://github.com/obiefy/api-response) - Simple and ready to use API response wrapper for Laravel.
+- [JoeVue](https://github.com/Miaababikir/JoeVue) - CLI tool help you create your vue file easily.
+
+## UI Libraries & Components
+
 - [Empathy](https://github.com/the94air/empathy) - A Lightweight Admin Panel Components With Tailwindcss & Vue.
 - [Meraki UI](https://github.com/merakiui/website) - Beautiful Tailwindcss components that support RTL out of the box & fully responsive.
 - [Tailwind Copy](https://github.com/the94air/twcopy) - Easily copy Tailwindcss colors and shadows.
 - [Tailwind Builder](https://github.com/Miaababikir/Tailwind-Builder) - Helps you to scaffold your forms structure without the need to repeat yourself.
-- [JoeVue](https://github.com/Miaababikir/JoeVue) - CLI tool help you create your vue file easily.
-- [Odoo-Intellij Plugin](https://github.com/firebitsnet/odoo-intellij) - Its a JetBrains's IDE Plugin that adds Support for developing Odoo applications and modules with IntelliJ IDEA and PyCharm IDEs.
+
+## IDE Extensions
+
+- [Odoo-Intellij Extension](https://github.com/firebitsnet/odoo-intellij) - Its a JetBrains's IDE Plugin that adds Support for developing Odoo applications and modules with IntelliJ IDEA and PyCharm IDEs.
+- [Laravel-snip Extension](https://marketplace.visualstudio.com/items?itemName=khatabwedaa.laravel-snip) - Simple vscode Extension for Laravel routes.
```

Comment:
Kindly update the link to the Open Source repo :)

Reply:
i deleted this repo :laughing: but i will remove it :+1: